### ISSUE TITLE
Plug-in group not saving

### ISSUE THREAD

**Issue by kmaustral** [Mocca101/obsidian-plugin-groups#25]
I save a selection of plug-ins for a group, but when I re-start Obsidian, all the plug-ins are activated. The group is set to load at start up. 

Is it possible that sync is re-setting the group?

**Comment by Mocca101** [comment]
Hm, it shouldn't, would you mind sharing your plugin groups settings?
And when you say all plugins are activated do you mean through obsidian or through plugin groups?

**Comment by kmaustral** [comment]
Thanks for your reply. Does this help?

![image](https://user-images.githubusercontent.com/73867063/224573528-377aae1a-5e2f-438b-8bea-73d9d2f634e3.png)

**Comment by Mocca101** [comment]
Could you clarify something more for me: When you say all the plugins are activated do you mean all plugins are added to the group "Normal"?
If not, what exactly do you mean with activated?

**Comment by kmaustral** [comment]
Yes. They are added to the Normal group. But saying that, the settings now seem to have stabilised. I do wonder if it something that can be influenced by sync. For anyone experiencing similar problems, the best solution is to persist and re-save settings until it stabilises.

**Comment by Mocca101** [comment]
Strange, I don't think it should have anything to do with sync, but I'm glad to hear it works now. In case it stops working again, feel free to return this issue.

**Comment by kmaustral** [comment]
The saving no longer works, I'm afraid. I edit the plug-in group by de-selecting particular plugins and then press "save". But whenever I re-start Obisidian, all the plug-ins are selected. It has the option for "load at start up".
